cmd/kops/create_cluster.go

```diff
@@ -1079,6 +1084,10 @@ func RunCreateCluster(f *util.Factory, out io.Writer, c *CreateClusterOptions) e
 		cluster.Spec.SSHAccess = c.SSHAccess
 	}
 
+	if c.NoSSHKey {
+		cluster.Spec.NoSSHKey = c.NoSSHKey
```

Comment:
Why do you use the if here? If not provided it defaults to false but should be set nonetheless.

Reply:
Do we want to write the default bool value of "false"? My idea was to not clutter the configuration if it is not necessary.
Might relate to you previous point regarding whether the bool needs to be stored in the config or not?

Reply:
It is related. The thing is if you want to store it in the cluster spec giving it a value of either true or false would be good. 

In this specific if what will happen is: 
If the value is true: cluster spec will state true 
If the value is false: cluster spec will be nil 
If you don’t specify it on the command: cobra will default to false -> cluster spec will state nil